### DISCUSSION TITLE
Mi Band WatchFace Editor GUI

### DISCUSSION THREAD

**Discussion by Johnson070** [Johnson070/MiBand-5-watchface-editor#34] [General]
1.0.1.8 (2021-06-09)

- Fixed Circle Scale bug

- Fixed alarm on/off add image

- Fixed oneline date

- Fixed bug number size 0

- Fixed ImageSufficCN2 to ImageSuffixCN2 (lol)

Thank you for downloading the program.

If you find a bug, please write to github or telegram: @vveber

<hr /><em>This discussion was created from the release <a href='https://github.com/Johnson070/MiBand-5-watchface-editor/releases/tag/1.0.1.8'>Mi Band WatchFace Editor GUI</a>.</em>

**Comment by drewdus42** [comment]
cant download on chrome, getting a virus detected message

**Reply by anicetoclucas** [reply]
Me too

**Comment by Tbear2good** [comment]
Can you please make a tutorial on how to use this please?

**Reply by Johnson070** [reply]
https://youtu.be/r91-eIF-DBs

English subtitles
